Fix OpenJDK installation instructions for Mac OS X homebrew

chmreid opened the pull request:
**Description**

This updates the install instructions for OpenJDK on Mac using homebrew. Complements #1342 which covered Mac install instructions for the OpenJDK-provided tarball.

**Issue**
dockstore/dockstore#4486
https://ucsc-cgl.atlassian.net/browse/DOCK-1925

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization (**NOTE:** this modifies markdown contained in strings, but doesn't change how the markdown is displayed.)
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain

coverbeck reviewed (approved):
Should this target the hotfix?

denis-yuen reviewed (commented):
Agreed, please cherry-pick to hotfix to match issue

chmreid commented:
Rebased the pull request branch onto `hotfix/2.8.3`, and kept the branch name the same. Updated the pull request base branch to be `hotfix/2.8.3` instead of `develop`. Did a force-push and the PR changes made it through intact.
